object-cache.copy.php

```diff
@@ -0,0 +1,55 @@
+
+<?php
+/**
+ * Plugin Name: Plugin Check Object Cache Drop-In
+ * Plugin URI: https://github.com/WordPress/plugin-check
+ * Description: Plugin check drop-in to setup the test environment early. This is not a real object cache drop-in and will not override other actual object cache drop-ins.
+ * Version: 1
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ *
+ * Object cache drop-in from Plugin Check.
+ *
+ * This drop-in is used, admittedly as a hack, to be able to setup the
+ * WordPress environment as early as possible. Once a plugin is loaded, it is
+ * too late to configure the test environment.
+ *
+ * This file respects any real object cache implementation the site may already
+ * be using, and it is implemented in a way that there is no risk for breakage.
+ *
+ * If you do not want the Plugin Check plugin to place this file and thus be
+ * limited to static checks, you can remove this file and set the following
+ * constant (e.g. in wp-config.php):
+ *
+ *     define( 'WP_PLUGIN_CHECK_DISABLE_OBJECT_CACHE_DROPIN', true );
```

Comment:
I don't think we need that here, as we're not as "aggressively" placing our drop-in as the PL plugin does (which I still need to revise). I'd prefer if we remove this constant therefore.